PantheonCMD/pcyamlchecks.py

```diff
@@ -58,9 +58,9 @@ def get_yaml_errors(yaml_schema, yaml_doc):
     v.validate(yaml_doc, yaml_schema)
 
     if v.errors:
-        print("FAIL: there is an error in your yaml file:")
-        for key in v.errors.keys():
-            print("\n\t'{}' {}".format(key, ', '.join(str(item) for item in v.errors[key])))
+        print("ERROR: there is an error in your yaml file:")
```

Comment:
If this changes to 'ERROR', perhaps it makes more sense to phrase the message as 'Your YAML file contains an error.' or something similar.